src/plugins/funcs/funcs.cpp

```diff
@@ -44,7 +44,7 @@ BOOL APIENTRY DllMain(HMODULE hModule, DWORD ul_reason_for_call, LPVOID lpReserv
 		plugin = new Funcs();
 
 		char szPath[MAX_PATH];
-		GetModuleFileName(hModule, szPath, MAX_PATH);
+		GetModuleFileNameA(hModule, szPath, MAX_PATH);

```

Comment:
Speaking to each similar reference. If you use GetModuleFileNameA you are enforcing an ASCII implementation, but GetModuleFileName() would allow a flexible approach. If the plugin is built with Unicode or not, GetModuleFileName() accommodates. You can hardcode the *A() functions, but again, it is not as flexible if we are modernizing the application.

Reply:
the thing is that SetPluginFullPath takes an ASCII string as input, and SetPluginFullPath cannot be changed without breaking the plugins ABI. 
I could have used GetModuleFileName, but I would have needed to convert the wstring back to ascii string. I don't know if there is any issue with using directly GetModuleFileNameA and bypassing the macro.